esptool.py

```diff
@@ -18,6 +18,8 @@
 # this program; if not, write to the Free Software Foundation, Inc., 51 Franklin
 # Street, Fifth Floor, Boston, MA 02110-1301 USA.
 
+from __future__ import print_function, unicode_literals, division
+
```

Comment:
I'm not so sure about unicode literals. I don't know the codebase well enough, but in a previous 2/3 compat project, we ran into issues with lots of os level APis not accepting unicode, so I'd be careful with it.  Some more discussion is here: https://github.com/PythonCharmers/python-future/issues/22